tests/concurrent_test.go

```diff
@@ -13,6 +13,9 @@ import (
 )
 
 func TestProxy_Concurrency(t *testing.T) {
+	t.Log("### Starting TestProxy_Concurrency ###")
+	defer t.Log("### Stopping TestProxy_Concurrency ###")
```

Comment:
Why do we need these log lines?

Reply:
Crud, I thought I had cleared all these up. These are leftovers from me tracing where the leaked goroutines were coming from. I'll go ahead and clear these up.